osg_configure/configure_modules/infoservices.py

```diff
@@ -340,10 +347,17 @@ def _resourcecatalog_location(self):
             return match.group(1)
 
 
-def ensure_valid_user_vo_file(using_gums, logger=utilities.NullLogger):
+def ensure_valid_user_vo_file(using_gums, gums_host=None, logger=utilities.NullLogger):
     if not validation.valid_user_vo_file(USER_VO_MAP_LOCATION):
         logger.info("Trying to create user-vo-map file")
         if using_gums:
+            try:
+                user_vo_file_text = gums_supported_vos.gums_json_user_vo_map_file(gums_host)
+                open(USER_VO_MAP_LOCATION, "w").write(user_vo_file_text)
+                return True
+            except exceptions.ApplicationError, e:
+                self.log("Could not query GUMS server via JSON interface: %s" % e, level=logging.DEBUG)
```

Comment:
So, this exception just indicates that the gums server is not a new enough version that has the newer json interface, in which case they still need to use `gums-host-cron` from `gums-client` ... But sure we can make it a warning.